bridgetown-core/lib/bridgetown-core/commands/concerns/actions.rb

```diff
@@ -77,23 +77,34 @@ def apply_from_url(url)
 
       private
 
+      def remote_file(arg)
+        if arg.end_with?(".rb")
+          arg.split("/").yield_self do |segments|
+            arg.sub!(%r!/#{segments.last}$!, "")
+            segments.last
+          end
+        else
+          "bridgetown.automation.rb"
+        end
+      end
+
       # TODO: option to download and confirm remote automation?
       def transform_automation_url(arg)
         return arg unless arg.start_with?("http")
 
-        remote_file = if arg.end_with?(".rb")
-                        arg.split("/").yield_self do |segments|
-                          arg.sub!(%r!/#{segments.last}$!, "")
-                          segments.last
-                        end
-                      else
-                        "bridgetown.automation.rb"
-                      end
+        remote_file = remote_file(arg)
```

Comment:
I generally prefer to name local variables differently than methods so there's no confusion. Not sure what makes sense here…would love suggestions.

Reply:
Honestly, I only did this to make rubocop happy. Could name it something like `find_remote_filename(arg)` and then do 

`remote_file = find_remote_filename(arg)`

Reply:
How about `determine_remote_filename`

Reply:
sounds good, ill make it happen.